Cargo.toml

```diff
@@ -3,9 +3,9 @@ categories = ["embedded", "no-std"]
 name = "enc424j600"
 description = "Embbeded Rust Ethernet driver for ENC424J600 Ethernet controller with SPI interface"
 authors = ["Harry Ho <hh@m-labs.hk>", "Dip Cheung <dc@m-labs.hk>"]
-version = "0.2.0"
+version = "0.3.0"
 keywords = ["ethernet", "eth", "enc424j600", "stm32", "stm32f4xx"]
-repository = "https://git.m-labs.hk/M-Labs/ENC424J600"
+repository = "https://github.com/quartiq/enc424j600"
```

Comment:
Let's keep the repository link as is. Assuming cargo doesn't complain. I'll push there later.